config/locales/es.yml

```diff
@@ -0,0 +1,11 @@
+es:
+  ibandit:
+    invalid_country_code: "'%{country_code}' no es un código de país ISO 3166-1 válido"
+    invalid_check_digits: "Los dígitos de control no han pasado la verificación de módulo. Se esperaba: '%{expected_check_digits}', recibido '%{check_digits}'."
+    invalid_length: "La longitud no casa con la specificación SWIFT (Se esperaba %{expected_length} símbulos, se ha recibido %{length})"
```

Comment:
I'm pretty sure 'casa' is a false friend (http://en.wikipedia.org/wiki/False_friend). Could use 'prevista' since we already use 'esperada'